timeless/employees/views.py

```diff
@@ -1,48 +1,62 @@
 """Employees views module.
-@todo #57:30min Continue implementing Create, Edit and Delete views once
- generic views from #173 are implemented. Templates are already provided below,
- so uncomment and modify - create templates and write ITs to verify behaviour
 @todo #57:30min Continue implementing List to enable sorting and filtering
  for every column. For this, probably there will be a need to create a new
  generic view that all other List views will extend. This generic view should
  use GenericFilter implemented in #317.
 """
 from flask import Blueprint
-from timeless.views import ListView
+from timeless.employees.forms import EmployeeForm
+from timeless import views
 from timeless.employees.models import Employee
 
 bp = Blueprint("employee", __name__, url_prefix="/employees")
 
 
-# class Create(CreateView):
-#     """Create employee"""
-#     template_name = "employees/create_edit.html"
-#     form_class  = EmployeeForm
-#     model = Employee
+class Create(views.CreateView):
+    """Create employee"""
+    template_name = "employees/create_edit.html"
+    form_class = EmployeeForm
+    model = Employee
+    success_view_name = "employee.list"
 
+    def get(self, **kwargs):
+        context = self.get_context(action="create")
+        return self.render_to_response(context)
 
-# class Edit(UpdateView):
+"""
+@todo #348:30min Continue Edit view when generic view UpdateView would be implemented.
+Template "employees/create_edit.html" has already provided, so uncomment and modify. 
+Write IT to verify behaviour.
+"""
+# class Edit(views.UpdateView):
 #     """Update employee"""
 #     template_name = "employees/create_edit.html"
 #     form_class  = EmployeeForm
 #     model = Employee
 
 
-# class Delete(DeleteView):
-#     """Delete employee
-#     Deletes employee using id and redirects to list page
-#     """
-#     form_class  = EmployeeForm
-#     model = Employee
+class Delete(views.DeleteView):
```

Comment:
@VladimirRyzhkov pls take at class attrs. here is a lot of extra

Reply:
@vladarefiev Vlad, I dropped one extra attribute. Others are needed for current and future (for #312) implementations.